virt/virtual_machines/virt-triggering-vm-failover-resolving-failed-node.adoc

```diff
@@ -0,0 +1,30 @@
+[id="virt-triggering-vm-failover-resolving-failed-node"]
+= Triggering virtual machine failover by resolving a failed node
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-triggering-vm-failover-resolving-failed-node
+toc::[]
+
+If a node fails and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[machine health checks] are not deployed on your cluster, virtual machines (VMs) with `RunStrategy: Always` configured are not automatically relocated to healthy nodes. To trigger VM failover, you must manually delete the `Node` object.
+
+[NOTE]
+====
+If you installed your cluster by using xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[installer-provisioned infrastructure] and you properly configured machine health checks:
```

Comment:
Just a thought, since there's an important distinction made here that this procedure is not relevant to these clusters. Is it worth pulling this out of the note so that it's more visible?

Reply:
It feels right to keep this info isolated from the rest of the assembly, and I'm not sure how to do that other than by putting it in a note. I get what you're saying, though.